src/android/aidl/birthday-service.md

```diff
@@ -0,0 +1,3 @@
+# Birthday Service Tutorial
+
+We doing an example! We're going to create a Birthday Service that does... something.
```

Comment:
Assuming this is going to be improved? Title slides should serve some purpose (or we can skip it).

Reply:
Yup! Wording has been improved. It's still a quick slide, but now it at least introduces the section.